### PR TITLE
setting appended CC to payload

### PR DESCRIPTION
**Pull request by azhamn**

### CODE REVIEW

File: com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
+            <header expression="get-property('jwt-header')" name="x-jwt-assertion" scope="transport"/>
+        </then>
+        <else/>
+    </filter>
+    <header name="Content-Type" scope="transport" value="application/json"/>
+    <header name="Accept" scope="transport" value="application/json"/>
+    <header name="Accept-Charset" scope="transport" value="UTF-8"/>
+    <!-- remove below comment for production -->
+    <header name="sandbox" scope="transport" expression="get-property('uri.var.onBehalfOf')"/>
+    <log level="custom" xmlns="http://ws.apache.org/ns/synapse">
```
Comment (Buddhima) [inline com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml]: remove unnecessary logs

File: com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
+    <header expression="get-property('ENDPOINT')" name="To" scope="default"/>
+
+    <filter regex="true" source="get-property('isUserInfoEnabled')">
+        <then>
+            <enrich>
+                <source clone="true" property="InitialMessage" type="property"/>
+                <target action="replace" type="body"/>
+            </enrich>
+            <header expression="get-property('auth-header')" name="Authorization" scope="transport"/>
+            <header expression="get-property('jwt-header')" name="x-jwt-assertion" scope="transport"/>
```
Comment (Buddhima) [inline com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml:13]: is this needed?

File: com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml [hunk 3]
```diff
@@ -1,38 +1,37 @@
...
+        </then>
+        <else/>
+    </filter>
+    <header name="Content-Type" scope="transport" value="application/json"/>
```
Comment (Buddhima) [inline com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml:17]: The headers can be set via "messageType" property

File: com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml [hunk 4]
```diff
@@ -1,38 +1,37 @@
...
+    <header name="Accept" scope="transport" value="application/json"/>
+    <header name="Accept-Charset" scope="transport" value="UTF-8"/>
+    <!-- remove below comment for production -->
+    <header name="sandbox" scope="transport" expression="get-property('uri.var.onBehalfOf')"/>
```
Comment (Buddhima) [inline com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml]: reason?

File: com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml (hunk 1 above)
Comment (shazni) [reply com.wso2telco.dep.hub.paymentapi/com.wso2telco.dep.hub.paymentapi.synapse/src/main/synapse-config/sequences/chargeRequestSeq.xml]: yes logs should be removed

**Pull request merged**